client/src/components/achievements/achievementsOnboarding.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable react/button-has-type */
+/* add new, delete, edit */
+import React, { useState } from 'react';
+import { Steps } from 'intro.js-react';
+import 'intro.js/introjs.css';
```

Comment:
prefer importing css after all the js has been imported, you can move this import down

Reply:
Done :)